docker/ci/install-pips.bash

```diff
@@ -38,3 +38,5 @@ pip install --quiet \
     PyUtilib==5.8.0 \
     pyomo==5.6.9
 
+# NOTE: glpk-utils is a required solver package of Pyomo.
+apt-get -qq install -y glpk-utils
```

Comment:
Put this line to `docker/dev/install.sh` ?

Reply:
Great idea! Done.